pkg/accounting/accounting.go

```diff
@@ -987,7 +992,8 @@ func (a *Accounting) NotifyPaymentSent(peer swarm.Address, amount *big.Int, rece
 	currentBalance, err := a.Balance(peer)
 	if err != nil {
 		if !errors.Is(err, ErrPeerNoBalance) {
-			a.logger.Error(err, "notify payment sent; failed to load balance")
+			a.logger.Debug("notify payment sent; failed to persist balance", "error", err)
+			a.logger.Error(nil, "notify payment sent; failed to persist balance")
```

Comment:
If you put the err in the logger.Error invocation, then you don't need the redundant logger.Debug since it adds no additional information.

Reply:
comment added below